.env_SAMPLE

```diff
@@ -179,6 +179,18 @@ export WATCHMAN_TOKENS=''
 #export NEW_RELIC_LICENSE_KEY=<new relic license key>
 #export NEW_RELIC_APP_NAME="cf.gov <your username here> python"
 
+############################################################################
+# SAML2 Authentication for local testing
+#
+# If testing our SAML2 authentication support, these variables are required,
+# as is running with the production settings.
```

Comment:
Worth including a link here [to the docs](https://djangosaml2.readthedocs.io/contents/setup.html#configuration)?

Reply:
Done in e765fc6.